src/test/regress/expected/pg12.out

```diff
@@ -361,8 +361,60 @@ SELECT DISTINCT y FROM test;
  25
 (1 row)
 
+-- non deterministic collations
+CREATE COLLATION case_insensitive (
+	provider = icu,
+	locale = 'und-u-ks-level2',
+	deterministic = false
+);
+\c - - - :worker_1_port
```

Comment:
We can use `run_command_on_master_and_workers` defined in `multi_test_helpers.sql`.